### PR TITLE
add sidebar component

### PR DESCRIPTION
**Pull request by danielrus**

### CODE REVIEW

**Review by sandorbanvitagroup — DISMISSED** [review]

File: src/app/shared/edge-panel/edge-panel.component.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          field: 'caze',
+          value: {
+            uuid: this.scopeId,
+          },
+        },
+      ]
+    );
+
+    this.breakpointObserver
+      .observe(['(max-width: 1024px)', '(min-width: 1680px)'])
```
Comment (adriandumavitagroup) [inline src/app/shared/edge-panel/edge-panel.component.ts]: maybe some constants here for the max width

File: src/app/shared/sidebar/sidebar.component.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+})
+export class SidebarComponent implements OnInit {
+  mode: MatDrawerMode = 'side';
+  position: 'start' | 'end' = 'end';
+  opened = true;
+
+  constructor(public breakpointObserver: BreakpointObserver) {}
+
+  ngOnInit(): void {
+    this.breakpointObserver.observe(['(max-width: 1024px)']).subscribe((state) => {
```
Comment (adriandumavitagroup) [inline src/app/shared/sidebar/sidebar.component.ts]: constant?

File: src/app/shared/edge-panel/edge-panel.component.ts (hunk 1 above)
Comment (danielrus) [reply src/app/shared/edge-panel/edge-panel.component.ts]: added constants for all breakpoints

File: src/app/shared/sidebar/sidebar.component.ts (hunk 2 above)
Comment (danielrus) [reply src/app/shared/sidebar/sidebar.component.ts]: added constants for all breakpoints

**Review by sandorbanvitagroup — APPROVED** [review]

**Pull request merged**